### PR TITLE
First capstone project

### PR DESCRIPTION
**Pull request by kkasaija**
First capstone project pull request.
I have added a link to the [Online Demo](https://kasaija-kenneth.github.io/) page. Please check it out.
Also included in this pull request is a link to [Loom overview video link](https://www.loom.com/share/b5b6f290eb474ccf8c31963a4e6046f3) where I highlighted the project features and layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                <img src="./images/ignite.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Resources</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
```
Comment (Chimwemwe-127001) [inline index.html]: - [ ] As the requirements suggest, make sure you follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
  
Figma design 
image

Your design
FireShot Capture 002 - East Africa Conference - raw githack com

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @Kasaija-Kenneth 

Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlight :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

## Required Changes ♻️

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+          <div class="d-grid gap-2 col-8 mx-auto mt-5">
+            <button class="btn btn-primary" type="button">Book a Seat</button>
+          </div>
+          <div class=" d-grid gap-2 col-8 mx-auto mt-3 p-3">
+            <h3 class="lead">Guest speakers</h3>
+          </div>
```
Comment (mwanawabangona) [inline index.html]: - [ ] Amazing work so far 👏 
- [ ] Book a seat is supposed to be moved at Guest speakers
- [ ] And guest speakers is moved insider the speakers section
- [ ] see screenshots below

`your tenplate`
image

`original design`
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/exhibit.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Exhibition</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
```
Comment (mwanawabangona) [inline index.html]: - [ ] kindly leave some space between the cards
`your desing`
image

`figma design`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi, @Kasaija-Kenneth 

Great work on making the changes requested by a previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @Kasaija-Kenneth , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Thanks to everybody that contributed to see this project a success. The code reviewers and other collaborators.

**Pull request merged**